lib/mutant/runner/sink.rb

```diff
@@ -49,6 +49,20 @@ def result(mutation_result)
 
     private
 
+      # Test if an evil failure has been reported
+      #
+      # @return [Boolean]
+      def evil_failure?
+        !status.subject_results.all?(&:success?)
```

Comment:
I'd be okay to have a `FailurePredicate` mixin that defines `#failure?` as the negation of `#success?`. But for consistency we should only ever concretely implement `#success?`.